torchvision/prototype/models/squeezenet.py

```diff
@@ -30,9 +30,11 @@ class SqueezeNet1_0_Weights(WeightsEnum):
         transforms=partial(ImageNetEval, crop_size=224),
         meta={
             **_COMMON_META,
+            "min_size": (21, 21),
             "num_params": 1248424,
             "acc@1": 58.092,
             "acc@5": 80.420,
+
```

Comment:
Remove this extra line which causes the python linter to [fail](https://app.circleci.com/pipelines/github/pytorch/vision/14077/workflows/041f0a9f-c9f3-4053-b9ab-78105f4cb784/jobs/1133128).